src/nifgen/examples/nifgen_script.py

```diff
@@ -2,52 +2,43 @@
 
 import argparse
 import nifgen
-import numpy as np
-from scipy import signal
+
+import math
+import random
 import sys
 import time
 
+# waveform size should be a multiple of the quantum, which is 4 or 8, for some devices
+# some older devices also have a 256 sample minimum waveform size
 number_of_points = 256
 
 
 def calculate_sinewave():
```

Comment:
TBH I think having functions for creating the waveforms is overkill.
A well named variable to hold it is self-documenting enough.

Reply:
replaced the functions with CONSTANTS.